client/blocks/get-apps/apps-badge.jsx

```diff
@@ -39,6 +43,10 @@ const APP_STORE_BADGE_URLS = {
 		src:
 			'https://play.google.com/intl/en_us/badges/images/generic/{localeSlug}_badge_web_generic.png',
 		tracksEvent: 'calypso_app_download_android_click',
+		storeLink: ( referrer ) =>
+			`https://play.google.com/store/apps/details?id=org.wordpress.android&referrer=utm_source%3D%${ referrer }%26utm_medium%3Dweb%26utm_campaign%3Dmobile-download`,
```

Comment:
We were using a different `utm_campaign` before.

```suggestion
			`https://play.google.com/store/apps/details?id=org.wordpress.android&referrer=utm_source%3D%${ referrer }%26utm_medium%3Dweb%26utm_campaign%3Dmobile-download-promo-pages`,
```

Reply:
Damn! Will do it.